autoinstall/FreeBSD/14/installerconfig

```diff
@@ -62,7 +62,13 @@ env ASSUME_ALWAYS_YES=YES pkg update -f > /dev/ttyu0
 
 # We install packages from ISO image
 # Different packages between the 32bit image and 64bit image
+machtype=$(env | grep MACHTYPE)
+{% if machtype.find('64') != -1 %}
 packages_to_install='bash sudo xorg kde5 xf86-video-vmware'
```

Comment:
I saw `sddm` would not be installed for FreeBSD 13.x 32bit. Shall it be removed from FreeBSD 14.x 32bit installation at below?

Since 32bit FreeBSD won't have desktop environment, we can also consider installing `open-vm-tools-nox11` instead of `open-vm-tools` on FreeBSD 13.x 32bit and FreeBSD 14.x 32bit. What do you think?